cyences_app_for_splunk/default/data/ui/views/cs_vpn_reports.xml

```diff
@@ -197,9 +197,40 @@
         </search>
         <option name="count">5</option>
         <option name="refresh.display">progressbar</option>
+        <option name="drilldown">row</option>
         <option name="rowNumbers">true</option>
         <drilldown>
-          <link target="_blank">search?q=%60rw_vpn_indexes%60%20TERM($row.User$)%20user=$row.User$%20TERM($row.SourceIP$)%20src_ip=$row.SourceIP$&amp;earliest=$time.earliest$&amp;latest=$time.latest$</link>
+          <link target="_blank">search?q=%60cs_vpn_indexes%60%20TERM($row.User$)%20user=$row.User$%20TERM($row.SourceIP$)%20src_ip=$row.SourceIP$&amp;earliest=$time.earliest$&amp;latest=$time.latest$</link>
+        </drilldown>
+      </table>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>Successful Session</title>
+      <table>
+        <search>
+          <query>`cs_vpn_indexes` dest_category="vpn_auth" action="success" $User$
+| stats values(src) as src_ip, values(private_ip) as private_ip by action, user, _time 
```

Comment:
Why don't we just use table? as we are doing group by _time, stats is not required.